libraries/adaptive-expressions/src/functionUtils.internal.ts

```diff
@@ -9,7 +9,7 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import { Constant } from './constant';
-import * as lodash from 'lodash';
+import { sortBy as lodashSortBy } from 'lodash';
```

Comment:
I think this will be more effective: https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

Feel free to verify that this gives the expected package size reductions.